apps/biobin/Readme.developer.md

```diff
@@ -1,14 +1,47 @@
+  - BioBin Summary Script - used only if the user would like to use older versions of the script that generates the summary output file, should be ignored in most cases.
 
-Ritchie Lab Software:/LOKI/*.db
-	A LOKI knowledge database whose filename indicates the date on which it was built.
+## Usage of the app
+Once you upload all the input files, go to settings where all the available options are listed. Unless you have a reason to change it, keep the "LOKI Knowledge Database" selection as is (the most recent LOKI version is selected by default). If you’d like to do binning followed by an association test, select one of the desired statistical tests: linear regression, logistic regression, wilcoxon test, SKAT-linear (SKAT for continuous phenotypes), or SKAT-logistic (SKAT for categorical phenotypes). In the window with "BioBin arguments" you should include options that you would like to use (specific to the type of analysis that you would like to run). All available options are listed in the manual.
 
+## Examples of analysis type-specific arguments 
+(note - these are just examples; these options are not necessarily complete and should be modified according to user’s preferences):
+  - Gene binning analysis:
+  ```
+  --weight-loci Y --bin-pathways N --bin-regions Y  --bin-minimum-size 5 -F 0.01 -G 37
+  ```
+  - Pathway binning analysis:
+  ```
+  --weight-loci Y --bin-pathways Y --bin-regions N --bin-interregion N --bin-minimum-size 5 -F 0.01 -G 37
+  ```
+  - Gene binning analysis limited to only the putative functional variants using a role file:
+  ```
+  --weight-loci Y --bin-pathways N --bin-regions Y --bin-minimum-size 5 -F 0.01 -G 37 --bin-expand-roles Y --bin-expand-size 1
+  ```
+
+If the user would like to perform a permutation analysis to empirically assess the significance of the obtained results, the app (unlike the BioBin command-line application) can also be used to run a permutation analysis, in which the sample IDs in the phenotype file are being permuted. For that, the user needs to specify, in the "Permutation Count" window, how many permutations he/she would like to perform, and the app will re-run the BioBin analysis for each permuted phenotype file, keeping track of the p-values obtained from each permuted test. In addition to the regular output files described below, permutation analysis also outputs a `*permute-summary.tsv` file, which contains the empirical p-value (obtained based on individual p-values from the permuted tests) for each phenotype-bin combination tested (see below for details).
+
+## What does this app output?
+For detailed description of the output files listed below, please see the software manual.
+  - Bins report file (`*bins.csv`; one per phenotype)
+  - Summary file (`*summary.tsv`) - specific to the BioBin app, not available in the BioBin command-line application. This file contains data from the bins reports concatenated across all phenotypes.
```

Comment:
- Summary file (`*summary.tsv`) - specific to the BioBin app; not available in the BioBin command-line application. This file contains data from the bins reports concatenated across all phenotypes.